operator/v1/types_network.go

```diff
@@ -485,6 +487,11 @@ type IPsecConfig struct {
 }
 
 type IPsecExternal struct {
+	// State controls the node's external (aka NS) ipsec service state.
+	// +kubebuilder:validation:Enum="";Enabled;Disabled
```

Comment:
deafault=Disabled, means we can remove the empty string "" option

Reply:
This didn't work for me when the cluster starts. Somehow even with this default during the installation I got an error that value "" is not supported.

Reply:
not sure which didnt work
the default setting?
or the ""

assuming default works, I'm saying you can remove the ""..

Reply:
even with default=disabled, I got an error that "" is not supported. I am not sure how this works, but since the object is mandatory and is created by the system and not by the user, is is created with a value of "". My suggestion is that in the operator we will change "" to Disabled so for discoverability we will see only enable or disable but on the yaml file empty string is acceptable as valid input.